project.yaml

```diff
@@ -1311,6 +1311,38 @@ Layer:
     properties:
       minzoom: 13
     advanced: {}
+  - name: "admin-very-low-zoom"
+    id: "admin-very-low-zoom"
+    class: ""
+    geometry: "linestring"
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+            way,
+            admin_level
+          FROM planet_osm_roads l
+          WHERE boundary = 'administrative'
+            AND (admin_level IN ('0', '1', '2')
+              OR (admin_level IN ('3', '4')
+                AND EXISTS
+                  (SELECT 1
+                    FROM planet_osm_polygon p
+                    WHERE boundary = 'administrative'
+                      AND admin_level = '2'
+                      AND way_area > 9e+12
+                      AND ST_Within(ST_Centroid(l.way), p.way)
```

Comment:
This doesn't work.

Consider the yellow country, where black are admin level 2 boundaries and red is admin level 4. Assume that way_area of the yellow country is > 9e12

![image](https://cloud.githubusercontent.com/assets/1190866/11322105/117fa5d0-9090-11e5-9a44-11dec1450a97.png)

The red way is within the country, but its centroid at blue is outside. Another error would occur if white were a country with enough area and yellow were not.

I think `ST_Within(l.way, p.way)` would work.

Also, `l` and `p` table aliases differ from the place label table aliases.